pkg/docker/config/config.go

```diff
@@ -739,13 +752,27 @@ func stripScheme(url string) string {
 	return stripped
 }
 
+// normalizeRegistry converts the provided registry to contain only the hostname.
 func normalizeRegistry(registry string) string {
 	normalized := convertToHostname(registry)
-	switch normalized {
+	return convertDockerHost(normalized)
+}
+
+// stripRegistry removes known prefixes as well as suffixes from the registry.
+func stripRegistry(registry string) string {
```

Comment:
So the input is not a registry, otherwise there would be nothing to strip.

Reply:
This function does not exist any more.